src/app/modules/shared-components/components/state-widget/state-widget.component.scss

```diff
@@ -8,7 +8,7 @@
   display: flex;
   font-size: .9rem;
 
-  &.discovery {
+  &.Undefined {
```

Comment:
Use a stored variable which is the lowercase of type instead.

Reply:
fixed